client/account-recovery/reset-password/reset-password-confirm-form/style.scss

```diff
@@ -0,0 +1,50 @@
+.reset-password-confirm-form__title {
+	font-size: 20px;
+	font-weight: 600;
+	margin-bottom: 16px;
+}
+
+.form-button.reset-password-confirm-form__button {
+	width: 100%;
+	margin: 8px 0;
+}
+
+.reset-password-confirm-form__description {
+	margin: 4px 0;
+}
+
+.form-label.reset-password-confirm-form__text-input-label {
+	display: inline;
+	float: left;
+}
+
+.reset-password-confirm-form__show-hide-toggle {
+	display: inline;
+	float: right;
+	line-height: 16px;
+}
+
+.form-button.reset-password-confirm-form__button.generate-password-button {
+	line-height: 16px;
+}
+
+.form-button.reset-password-confirm-form__button.generate-password-button:before {
+	content: '\f427';
+	font: normal 16px/1 'Noticons';
```

Comment:
Since there is no `key` icon in Gridicons. @evilluendas and I had concluded that we should simply remove it :)